spec/services/bulk_ingest_service_spec.rb

```diff
@@ -257,10 +259,57 @@
         expect(resource.viewing_hint).to eq ["paged"] # brought in from figgy_metadata.json
         expect(resource.member_ids.length).to eq 2 # color.tif, gray.tif
         expect(resource.depositor).to eq ["tpend"]
+        expect(resource.title).not_to eq ["My Title"] # brought in from figgy_metadata.json
```

Comment:
This might be more clear if you test for the value that will overwrite it (looks like `Bible, Latin.`) and comment that it's overwriting the value that was in figgy_metadata.json

Reply:
Oh, copy line 307 here!